### ISSUE TITLE
Missions not to be canceled by parking

### ISSUE THREAD

**Issue by DynV** [endless-sky/endless-sky#4683]
**Is your feature request related to a problem? Please describe.**
I think it's illogical that missions need to be canceled when desiring to have the ship necessary for them to be parked temporarily; and I wish there would be a mechanism to include it (temporary parking not cancelling missions).

The issue that made me request the feature, is that I packed my missions together but by coincidence there's 1 planet that has a lot but is out of the way and not economically viable for the fleet to make a detour. I'm thinking what's the difference for the passengers & cargo between
-  following the flagship and
- getting parked a few days
while both of them would have the passengers & cargo at their destination at the _same time_. Imagine if this was a real situation and you asked the passengers if during your detour if they'd prefer them to stay on a planet a few days or stay in the ships; and of course the cargo can't care either way.

The temporary parking mechanism could even force the fleet to come back to the same system for the missions to resume to ensure the player is not taking advantage of taking a different route on his/her way back from the detour and ending up being better off than if the ships that are required for those missions would not have been parked. Or to be less strict, either the same system or one adjacent to it.

**Describe the solution you'd like**
Being allowed to pause missions while having the ships necessary for them to be parked. I see 2 ways of doing this, you either
- request to park some ships and the system let you know you need to pause enough missions that sums up to such & such number of passenger & cargo (as you need the unpaused missions to have enough of them), or you
- pause missions, and the system let you know up to how many passenger seats and cargo space can be parked.

**Comment by petervdmeer** [comment]
I think I understand the problem described in this feature request, but I don't think that pausing missions is the solution.

Some time ago I wrote some text for a "Standby" feature which I just now posted in #4690. I have the impression that the "Standby" feature might also solve the problem you describe in this feature request. Can you check if it indeed solves the problem described here?

**Comment by DynV** [comment]
It is not the same. The goal of this feature request is to save money; especially to go somewhere that is not economically advantageous otherwise. Let me make a crude drawing of what I mean
![endless_sky-issue_4683-example](https://user-images.githubusercontent.com/2823807/69834684-f3a67700-1209-11ea-8618-90b45e1bf845.png) in which the colors is in ascending level of profitability from green to red. My idea is to _temporarily_ park all/most ships at the bottom in either the yellow or green area, depending on how costly the fleet is to maintain, then land in the red area making as much profit as possible, then come back to get the remainder of the fleet by either landing on the same planet, or an adjacent one, and while on that planet unpark at least enough ships so none of the missions are canceled.

The advantage of this feature request (for the player) : Not having to pay for part of the fleet maintenance for part of the fleet that otherwise would necessitate cancelling a(some) mission(s).

The disadvantage: It require the remainder of the fleet that wasn't temporarily parked to come back to the same planet, or one adjacent to it, that if not having used such feature might have taken a more efficient path; in essence possibly having to make a detour just to get the temporarily parked ships.

**Comment by petervdmeer** [comment]
The problem you are describing here might actually be a game feature; if it is not economical to do transport missions in the yellow and green zones and at the same time do a transport mission in the red zone, then the player has to choose which of the missions he/she picks up and which missions he/she doesn't accept or cancels.

This type of choice is the reason that planets usually offer multiple small missions to many different destinations; a single mission by itself might just not be very profitable, but combining multiple small missions to roughly the same destination can add additional profit for a small or no additional daily costs.

**Comment by DynV** [comment]
This is a detailed legend for the image in my last post:
- _red_ is worth landing even for _quite a large_ fleet
- _yellow_ a _somewhat large_ fleet would find landing to be profitable
- _green_ a _medium fleet_ would make money by landing there
- _white_ only a _tiny fleet and perhaps a small one_ would make a profit landing there
Using the term of this legend, a _somewhat large fleet_ would not make money landing in green nor white; since having a detour in there only adds to the upkeep cost, a player should _ask him/herself if it's worth doing a detour to the red sector_.

About stacking missions, see for yourself
![EndlessSky-stacking_missions](https://user-images.githubusercontent.com/2823807/69894157-c204dd00-12e9-11ea-823b-b16090d0fefa.png)
I'm heading North-East.

**Comment by petervdmeer** [comment]
Wow, nice to see that you really have a lot of missions active in parallel. I however still feel that it does make sense that the player makes choices if combining missions doesn't result in a profit for the combination.

The player gets paid for transporting persons and cargo, so being able to just drop some persons and cargo on a planet and pick them up later doesn't really sound fitting to me. Why would a passenger not just book a flight with another captain that does have time for him/her earlier?

**Comment by DynV** [comment]
> Why would a passenger not just book a flight with another captain that does have time for him/her earlier?
For the same reason s/he stick around when I have many of the transports of my fleet do a--large--detour? ie: I have a deadly armament fleet and they don't have to pay for food nor accommodation on that detour.

**Comment by petervdmeer** [comment]
Closing this issue with the label unlikely; not being able to accept every mission and needing to make choices about which missions to accept together is a game-feature.
